### PR TITLE
Computational grid reordering

### PR DESCRIPTION
**Pull request by ldebek**

### CODE REVIEW

File: processing/results_analysis/gpkg_conversion.py [hunk 1]
```diff
@@ -130,13 +129,8 @@ def processAlgorithm(self, parameters, context, feedback):
             qml_path = safe_join(styles_dir, f"{table_name}.qml")
             if os.path.exists(qml_path):
                 layer.loadNamedStyle(qml_path)
-            context.temporaryLayerStore().addMapLayer(layer)
```
Comment (benvanbasten-ns) [inline processing/results_analysis/gpkg_conversion.py:133]: is the set "epsg_codes" on line 118 ever appended?

File: processing/results_analysis/gpkg_conversion.py [hunk 2]
```diff
@@ -47,7 +46,7 @@ def name(self):
         return "threedi_convert_gridadmin_to_gpkg"
 
     def displayName(self):
-        return self.tr("Convert gridadmin to GeoPackage")
+        return self.tr("Computational grid from gridadmin.h5 file")
```
Comment (benvanbasten-ns) [inline processing/results_analysis/gpkg_conversion.py:35]: Perhaps also updated shortHelpString()?

File: processing/results_analysis/grid_creation.py [hunk 3]
```diff
@@ -10,13 +11,17 @@
...
 )
 from threedigrid_builder import make_gridadmin, SchematisationError
+from threedigrid.admin.exporters.geopackage import GeopackageExporter
+from ThreeDiToolbox.processing.results_analysis.gpkg_conversion import Progress
+from ThreeDiToolbox.utils.utils import safe_join
 
 
 class ThreeDiGenerateCompGridAlgorithm(QgsProcessingAlgorithm):
-    """Generate a gridadmin.h5 file out of Spatialite database"""
+    """Generate a gridadmin.h5 file out of Spatialite database and convert it to GeoPackage."""
```
Comment (benvanbasten-ns) [inline processing/results_analysis/grid_creation.py]: Perhaps also add that several layers are added to the project?

**Comment by benvanbasten-ns** [comment]
Nice work. 

I noticed that there is some code duplication in grid_creation and gpkg_conversion. Both convert to a geopackage and add a subset of layers. You could consider moving this code (gpkg_conversion line 98-163) out of the ProcessingAlgorithms so both algorithms share this code.

**Comment by leendertvanwolfswinkel** [comment]
Our functional tester was really positive about it, so after @benvanbasten-ns comments have been dealt with, it can be merged with master

File: processing/results_analysis/gpkg_conversion.py (hunk 2 above)
Comment (ldebek) [reply processing/results_analysis/gpkg_conversion.py:35]: Done.

File: processing/results_analysis/gpkg_conversion.py (hunk 1 above)
Comment (ldebek) [reply processing/results_analysis/gpkg_conversion.py:133]: Fixed.

**Comment by ldebek** [comment]
@benvanbasten-ns I've refactored duplicated code bits.

**Pull request merged**